marketplace-ui/src/app/modules/product/product-detail/product-detail-version-action/product-detail-version-action.component.scss

```diff
@@ -73,7 +73,6 @@
 
 .btn,
 ::ng-deep .install-designer-button {
-  padding: 12px 32px;
```

Comment:
Hi Dinh , although It was fixed in designer but I break the UI of 2 buttons in the website